openreview/conference/builder.py

```diff
@@ -2257,12 +2257,15 @@ def set_submission_stage(
             email_pcs_on_desk_reject=True,
             author_names_revealed=False,
             papers_released=False,
-            readers=None
+            readers=None,
+            bidding_enabled=False
```

Comment:
We are having too many parameters to decide the readers of the notes: public, papers_release, readers and bidding _enabled. I wish we can merge this into one.

I would prefer to use readers param and don't add a new param. Also don't you need to check if there are SAC or AC in the conference? (I guess this was working like this before but we should fix it)

Reply:
Sounds good. 

submission_stage.get_readers() checks if the conference has SACs and ACs